crates/holochain_types/src/app/app_manifest/app_manifest_v0.rs

```diff
@@ -54,6 +54,20 @@ pub struct AppManifestV0 {
     #[serde(default)]
     #[builder(default)]
     pub allow_deferred_memproofs: bool,
+
+    /// URL of the bootstrap server to use for all Cells created
+    /// for this app. If not provided here, the conductor's
+    /// default bootstrap server will be used.
```

Comment:
This being different to the documentation below is a bit confusing. The conductor doesn't have a default bootstrap server until one is provided in the conductor config. I think the description below is clearer and should be used here.